src/test/java/core/basesyntax/RegistrationServiceTest.java

```diff
@@ -0,0 +1,135 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationException;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.Objects;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class RegistrationServiceTest {
+    private static final String VALID_STRING = "string";
+    private static final String ANOTHER_VALID_STRING = "another";
+    private static final String INVALID_STRING_WITH_LENGTH_ZERO = "";
+    private static final String INVALID_STRING_WITH_LENGTH_THREE = "qwe";
+    private static final String INVALID_STRING_WITH_LENGTH_FIVE = "asdfg";
+    private static final int VALID_AGE = 18;
+    private static final int ANOTHER_VALID_AGE = 22;
+    private static final int INVALID_AGE_SIX = 6;
+    private static final int INVALID_AGE_NINE = 9;
+    private static final int INVALID_AGE_SIXTEEN = 16;
+
+    private static StorageDao storage;
+    private static RegistrationService registrationService;
+    private static User user;
+
+    @BeforeEach
+    void setUp() {
+        registrationService = new RegistrationServiceImpl();
+        storage = new StorageDaoImpl();
+        user = new User();
+        user.setLogin(VALID_STRING);
+        user.setPassword(VALID_STRING);
+        user.setAge(VALID_AGE);
+    }
```

Comment:
storage is static. you should have one dao for the storage. and don't forget to clear storage between tests